### ISSUE TITLE
Windows 10 network install

### ISSUE THREAD

**Issue by rde100** [jfo8000/ScratchJr-Desktop#54]
1)	We want to use Scratch Jnr on our School PC’s however, the current installation of ScratchJr installs to the Local Appdata Folder of the current logged in user. This doesn’t work for us as it means we have to install the programme on every user's profile. A headache when you have over 300 users. Is there a fix for this so we can install the program once for all users?

2)	When installed on a user’s profile, It will work on one computer but when that user is on a different computer, Scratchjr will not load, even though it is installed on the computer.

**Comment by jfo8000** [comment]
Yeah someone else mentioned this - it's come up before.   I think really the only way round it is to build the MSI installer, which I have not attempted. :/

JFo
